packages/network-controller/CHANGELOG.md

```diff
@@ -6,6 +6,14 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [15.2.0]
+### Changed
+- Update @metamask/eth-json-rpc-middleware in network controller ([#1988](https://github.com/MetaMask/core/pull/1988))
+- move json-rpc-engine from merged-packages to packages ([#1895](https://github.com/MetaMask/core/pull/1895))
+- Bump @metamask/utils from 8.1.0 to 8.2.0 ([#1957](https://github.com/MetaMask/core/pull/1957))
+- Change `test` build scripts to only show output for failed tests by default ([#1949](https://github.com/MetaMask/core/pull/1949))
+- Re-enable import/no-named-as-default-member ([#1925](https://github.com/MetaMask/core/pull/1925))
```

Comment:
These changes are to lint rules and the test scripts; they have no impact upon the published package. We reserve this changelog just for things that affect the package, as it's for users of the package not maintainers.

```suggestion
```